Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/examples.txt

```diff
@@ -68,3 +68,4 @@ microsoft-atp-advanced-hunting-cover-up query_purpose=compromised_information us
 microsoft-atp-advanced-hunting-cover-up query_purpose=connected_devices username=demisto
 microsoft-atp-advanced-hunting-cover-up query_purpose=action_types username=demisto
 microsoft-atp-advanced-hunting-cover-up query_purpose=common_files  username=demisto
+microsoft-atp-live-response-cancel-action machine_action_id=11a86b87-12b8-423b-9e8d-9775ab2da78f comment="testing"
```

Comment:
I've added a command example to this file. Please change it if it's not accurate, and run the *generate-docs* command